### ISSUE TITLE
warning: instance variable @global_tags_formatted not initialized

### ISSUE THREAD

**Issue by mperham** [DataDog/dogstatsd-ruby#174]
I get lots of these warnings in my test suite (note two different line numbers):

```
/Users/mikeperham/.gem/ruby/2.7.2/gems/dogstatsd-ruby-5.0.0/lib/datadog/statsd/serialization/tag_serializer.rb:24: warning: instance variable @global_tags_formatted not initialized
/Users/mikeperham/.gem/ruby/2.7.2/gems/dogstatsd-ruby-5.0.0/lib/datadog/statsd/serialization/tag_serializer.rb:21: warning: instance variable @global_tags_formatted not initialized
```

**Comment by remeh** [comment]
Hey @mperham, thanks for having opened an issue. It should be fixed in 5.0.1

**Comment by remeh** [comment]
Closing this since it has been addressed in 5.0.1.
